sftp/1.3/modules/ROOT/edits/sftp-copy-move.adoc

```diff
@@ -0,0 +1,65 @@
+= Copy and Move Files with the SFTP Connector - Mule 4
+:page-aliases: connectors::sftp/sftp-copy-move.adoc
+
+Anypoint Connector for SFTP (SFTP Connector) Copy and Move operations enable you to copy and move files or directories on demand.
+
+While SFTP has a native concept of copying and moving, FTP does not. FTP Connector simulates these operations by downloading file content and writing it to a new location. Although all of this is done with concurrent streams that start writing and reading at the same time, copying and moving files does have a significant network overhead and should be used wisely. The SFTP Connector Move operation does not impact network overhead.
```

Comment:
I'm not sure about "having a concept" of actions (as opposed to an idea). Maybe...
```suggestion
While using SFTP includes copying and moving operations, FTP does not. FTP Connector simulates these operations by downloading file content and writing it to a new location, using concurrent streams that start writing and reading at the same time. Because these operations create significant network overhead, use them wisely. Note that the SFTP Connector Move operation does not impact network overhead.
```